kuma/static/js/components/user-signup/auth-modal.js

```diff
@@ -0,0 +1,40 @@
+(function() {
+    'use strict';
+
+    var authModalContainer = document.getElementById('auth-modal');
+
+    if (!window.mdn.modalDialog.showModal(authModalContainer)) {
+        return;
+    }
+
+    /* The React header takes just a couple of milliseconds to
+       long to load so, we need to wrap the code below in a 
+       setTimeout */
+    setTimeout(function() {
```

Comment:
Can be removed if not `async` loaded. 

PS. Tease me all you like but I still don't understand why all of this code stuff is done in ES5 vanilla JS and not in react.